test/CMakeLists.txt

```
@@ -2,7 +2,7 @@
 # Distributed under the Boost Software License, Version 1.0.
 # See accompanying file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt
 
-find_package(PythonInterp)
+find_package(Python3)
 if (PYTHONINTERP_FOUND)
```

This is not the same variable. And this requires CMake 3.15+.

---

```suggestion
if (Python3_FOUND)
```

This is the new variable.